IconSDK.Tests/TestRPC.cs

```diff
@@ -13,8 +14,22 @@ public class TestRPC
         public async Task Test_GetBalance()
         {
             var GetBalance = new GetBalance(API_URL);
+
+            // case 0: Successfull calls
+            // Get balance from EOA or score successfully
             var balance = await GetBalance.Invoke("hx0000000000000000000000000000000000000000");
-            Console.WriteLine($"Treasury : {balance.ToString()}");
+            Assert.IsInstanceOf<System.Numerics.BigInteger>(balance);
+            // No prefix
+            var balance2 = await GetBalance.Invoke("0000000000000000000000000000000000000000");
+            Assert.True(balance == balance2);
+
+            // case 1: when a param is wrong.
+            // Wrong hex format
+            Assert.ThrowsAsync<FormatException>(() => GetBalance.Invoke("hxWRONG"));
+            // Wrong prefix
+            Assert.ThrowsAsync<FormatException>(() => GetBalance.Invoke("0x0000000000000000000000000000000000000000"));
+            // Wrong number of bytes
+            Assert.ThrowsAsync<Exception>(() => GetBalance.Invoke("hx0000000000000000000"));
```

Comment:
Shouldn't it also throw a FormatException ?

Reply:
Yes you are right. I know exceptions are not specified. All exceptions are default exception(System.Exception) for now. I will modify them to custom or proper exceptions someday.
I have priorities for implementing features.
But your opinion is very reasonable. This SDK is unofficial and in developing.